### ISSUE TITLE
ImagePicker "import class " is not an option

### ISSUE THREAD

**Issue by Erodri07** [Dhaval2404/ImagePicker#313]
## Summary
![image](https://user-images.githubusercontent.com/71915136/204960413-8d2e5055-9944-422f-ab4d-b8feedb35450.png)
![image](https://user-images.githubusercontent.com/71915136/204960451-6b69cbaa-0f05-4112-9fcb-2470ed146599.png)
![image](https://user-images.githubusercontent.com/71915136/204960514-2bae5525-cbe4-4f49-8ab5-eec983c13f19.png)

I have followed the tutorial but ImagePicker is not being recognized as a class from this GitHub. What do I do? I am coding in Java, but I don't see how that could effect it.

**Comment by samety22** [comment]
Hi,

i have the same issue. Did you find a solution?

**Comment by jens-muenker** [comment]
Hi,

i have the same issue too. Did one of you two find a solution?

**Comment by samety22** [comment]
> Hi,
> 
> i have the same issue too. Did one of you two find a solution?

no. i switch to an other library - its better: the name was FishBun Image Picker

**Comment by jens-muenker** [comment]
> > Hi,
> > i have the same issue too. Did one of you two find a solution?
> 
> no. i switch to an other library - its better: the name was FishBun Image Picker

Okay, too bad but thanks for your answers. I copied the hole code into my project and made some changes. That's my current working solution but it's not easy to do it. - I need a image picker with aspect ratio. So I can't use your suggested library.
